WebRoot/js/zimbraMail/abook/view/ZmEditContactView.js

```diff
@@ -1071,15 +1070,21 @@ ZmEditContactViewImage = function(params) {
 		"<div style='width:48;height:48'>",
 			"<img id='",this._htmlElId,"_img' width='48' height='48'>",
 		"</div>",
-		"<div id='",this._htmlElId,"_badge' style='position:absolute;"
-        ,"bottom:",(AjxEnv.isMozilla ? -4 : 0), ";right:", (AjxEnv.isMozilla ? 3 : 0),"px'>"
+		"<div class='ZmEditContactImageBadge' id='",this._htmlElId,"_badge' style='position:absolute;"
+        ,"bottom:14px;right:14px'>"
 	].join("");
 	el.style.cursor = "pointer";
 
 	this._src = "";
+
+	/**
+	 * Using badgeEl to hold ContactImage icon.
+	 * Setting the imgEl opacity to 0 by default, 1 when image is available.
+	 * */
 	this._imgEl = document.getElementById(this._htmlElId+"_img");
-	this._imgEl.onload = AjxCallback.simpleClosure(this._imageLoaded, this);
+	this._imgEl.style.opacity = 0;
```

Comment:
We are toggling the opacity of the imgEl in the view based on some actions/ availability of image (in case of edit). I feel its better to handle opacity from one place i.e. js  and not having default from css and then changing it here.